app/wrappers/wakes/facebook_metrics_wrapper.rb

```diff
@@ -19,7 +19,7 @@ def share_counts
     {}.tap do |hash|
       individual_responses.map do |r|
         begin
-          hash[r['id']] = r.fetch('share').fetch('share_count')
+          hash[r['id']] = r.fetch('og_object').fetch('engagement').fetch('count')
```

Comment:
@gaganawhad is `r` a hash? Can we use `dig` here?

Reply:
@benhutton Unfortunately`dig` doesn't give us `KeyError`.

Reply:
@gaganawhad why not try this?

```
hash[r['id']] = r.dig('og_object', 'engagement', 'count')
if hash[r['id']].nil?
  Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
end
```